Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14
```

Comment:
@0xJem you didn't make this a dependency, correct? It looks like it's still an optional build alternative.